### PR TITLE
Complete counter

### PR DESCRIPTION
**Pull request by bumblebot1**
Added the div the shows the number of items left to complete.

### CODE REVIEW

File: public/main.js [hunk 1]
```diff
@@ -53,6 +58,14 @@ function reloadTodoList() {
             listItem.appendChild(completeButton);
             todoList.appendChild(listItem);
         });
+        if (count === 0) {
+            completeCounter.textContent = "";
+            completeCounter.className = "";
```
Comment (cuthullu) [inline public/main.js]: Is there a reason for removing the class name? Would it make more sense to just use css to hide the element when the count is 0?

File: public/main.js [hunk 2]
```diff
@@ -53,6 +58,14 @@ function reloadTodoList() {
             listItem.appendChild(completeButton);
             todoList.appendChild(listItem);
         });
+        completeCounter.className = "complete-count";
```
Comment (cuthullu) [inline public/main.js]: As this doesn't change now you can put it in the html

File: public/main.js [hunk 3]
```diff
@@ -53,6 +58,14 @@ function reloadTodoList() {
...
+        if (count === 0) {
+            completeCounter.style.visibility = "hidden";
```
Comment (cuthullu) [inline public/main.js]: Can't be styling in the JS. You'll have to add and remove a 'hidden' class on the object. I'd suggest looking at `String.split` and `Array.filter`

File: public/main.js [hunk 4]
```diff
@@ -53,6 +58,13 @@ function reloadTodoList() {
             listItem.appendChild(completeButton);
             todoList.appendChild(listItem);
         });
+        if (count === 0) {
+            completeCounter.style.visibility = "hidden";
```
Comment (kenneth-gray) [inline public/main.js]: Do be careful with `visibility` vs `display`

**Comment by cuthullu** [comment]
:shipit:

File: public/main.js [hunk 5]
```diff
@@ -53,6 +58,16 @@ function reloadTodoList() {
             listItem.appendChild(completeButton);
             todoList.appendChild(listItem);
         });
+        var classes = completeCounter.className.split(" ").filter(function(value) {
```
Comment (kenneth-gray) [inline public/main.js:61]: Suggestion:-

```
var classes = completeCounter.className
    .split(' ')
    .filter(function(value) {
        return value !== 'hidden';
    })
    .join(' ');
completeCounter.className = classes;
completeCounter.textContent = count;
if (count === 0) {
    completeCounter.className += ' hidden';
}
```

What do people think?

**Pull request merged**